test/index.spec.js

```diff
@@ -312,6 +312,43 @@ describe('Testing index', () => {
     }).timeout(10000);
   });
 
+  describe('Testing sorting', () => {
```

Comment:
Name? Maybe `Test sorting by score`?

Reply:
I did that change, and then changed it back to `Testing sorting` because of the added test